openrpc.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "MetaMask JSON-RPC API Reference",
     "version": "1.0.0",
-    "description": "This section provides an interactive reference for the MetaMask JSON-RPC API. This API builds on a set of standard Ethereum methods with MetaMask-specific enhancements, and is designed for seamless integration into dapps. You can try out individual RPC methods using the pre-configured examples or enter custom parameters. You can also use bookmarks to reuse and pre-fill any custom parameters."
+    "description": "This section provides an interactive reference for the MetaMask JSON-RPC API. The API builds on a set of standard Ethereum methods with MetaMask-specific enhancements, and is designed for seamless integration into dapps.\n\nView the JSON-RPC API methods by selecting them in the left sidebar. You can test the methods directly in the page, using the pre-configured examples or custom parameters. You can also use bookmarks to reuse and pre-fill any custom parameters.\n\nEach method may have one or more of the following labels:\n\n- **MetaMask** - These methods behave in ways specific to MetaMask, and may or may not be supported by other wallets.\n- **Snaps** - The methods are related to interacting with [Snaps](/snaps).\n- **Restricted** - These methods are restricted, which require requesting permission using [`wallet_requestPermissions`](/wallet/reference/wallet_requestpermissions).\n- **Mobile** - These methods are only available on MetaMask Mobile.\n- **Experimental** - These methods are experimental, and may be changed in the future.\n- **Deprecated** - These methods are deprecated, and may be removed in the future.\n- **Ethereum API** - These are standard Ethereum JSON-RPC API methods. See the [Ethereum wiki](https://eth.wiki/json-rpc/API#json-rpc-methods) for more information about these methods.\n\nYou can read more [about the MetaMask APIs](/wallet/concepts/apis)."
```

Comment:
> You can also use bookmarks to reuse and pre-fill any custom parameters.

Took me a second to realize you meant the browser's built in bookmarks (and not bookmarks built into the docs web app page), as in you can save the URL since it will include your prefilled params as you modify them in page.